### ISSUE TITLE
EDITOR: Snapping for gizmo T/R/S

### ISSUE THREAD

**Issue by nekitu** [OurMachinery/themachinery-public#15]

**Comment by niklas-ourmachinery** [comment]
Snapping is already available, but it is not super obvious. You enable it by double clicking a Tool in the UI to edit its properties.

**Comment by nekitu** [comment]
Ah ok, then prolly it should be more obvious in the toolbar or something, also some key modifiers to use the snap steps (Alt pressed while rotating will snap for example)

**Comment by niklas-ourmachinery** [comment]
Agreed... will have a think about this. Is Alt as hotkey for enabling snapping a common practice?

**Comment by nekitu** [comment]
Not sure if Alt, I have to check other editors and DCC tools, either way, some modifier key (aside Shift which is for cloning in your editor too)

**Comment by niklas-ourmachinery** [comment]
Yeah... it's a bit tricky... it's easy to run out of modifier keys :)

**Comment by chrisperrella** [comment]
Fwiw - Maya's snapping is bound to the `J` key.  Holding it will enable snapping on all transform tools.

**Comment by karl-zylinski-ourmachinery** [comment]
This was fixed at some point. Holding CTRL enables snapping.
